codespell_lib/data/linux-kernel.exclude

```diff
@@ -73,7 +73,7 @@ S: 1326 De Val-Brillant
 infinit:
 	bra	infinit
 the read on an 8-byte boundary (e.g., if you seeked an odd number of bytes
-#define ISNT		   12 
+#define ISNT		   12
```

Comment:
We should leave this as is, I think the line matching will fail otherwise:
https://github.com/torvalds/linux/blob/b24413180f5600bcb3bb70fbed5cf186b60864bd/arch/parisc/include/asm/perf.h#L31

Reply:
Fixed now.